apps/core/lib/core/services/shell/demo.ex

```diff
@@ -25,6 +25,9 @@ defmodule Core.Services.Shell.Demo do
   @lock "demo-projects"
   @max_count 3
 
+  @spec max_demo_count :: binary
```

Comment:
should it be `integer` instead of `binary`?

Reply:
I think it would be an integer.